{{cookiecutter.project_slug}}/clients/vue3/src/styles/buttons.css

```diff
@@ -6,5 +6,5 @@
 
 .btn--primary {
   @apply btn;
-  @apply bg-primary text-white font-bold py-2 px-4;   
-}
\ No newline at end of file
+  @apply text-white font-bold py-2 px-4;
```

Comment:
**Info:** With the upgrade to Tailwind V3, the background color applied here for `<button>` elements will not override the Tailwind default.

This is due to Tailwind applying default stylings to `<button>` elements as well as elements with attributes of
`[type='button'], [type='reset'], [type='submit']`.

Does not look to be a current "easy" fix, so for the time being - have just added the associated `bg-primary` class to the associated buttons on the Login and Signup views.

Relevant TailwindCSS discussion can be found here:
https://github.com/tailwindlabs/tailwindcss/discussions/5969